content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm/windows.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Single Step APM Instrumentation on Windows
+code_lang: windows
+type: multi-code-lang
+code_lang_weight: 30
+further_reading:
+  - link: /tracing/metrics/runtime_metrics/
+    tag: Documentation
+    text: Enable Runtime Metrics
+---
+
+<div class="alert alert-info"> Single Step APM Instrumentation on Windows is currently in preview.</div>
+
+## Overview
+
+With Single Step Instrumentation (SSI), you can automatically enable APM for your .NET Framework and .NET Core applications running on Windows IIS, using a single MSI command during the Datadog Agent installation. No separate SDK installation or code changes are required.
+
+## Enable APM on Windows
+
+To enable APM with Single Step Instrumentation on Windows:
+
+1. In the Datadog app, go to the Install Datadog [Windows Agent page][1] .
+1. Under Customize your observability coverage, toggle APM Instrumentation on.
+1. Copy and run the provided MSI install command on your Windows host.
+1. Restart the IIS applications you want instrumented. (You do not need to restart the entire IIS server.)
+
+Once installed, the Agent automatically loads the Datadog .NET SDK into supported application processes to enable distributed tracing.
+
+**Note:** Only .NET applications running in IIS are supported for Single Step Instrumentation on Windows.
```

Comment:
this seems like an important note to mention upfront. can we move it up to just after the h2?